add-ons/APPLICATION_TEMPLATE.md

```diff
@@ -0,0 +1,182 @@
+# New Project Project Application
+
+Please see the [proposals/new-project-addition-process.md](Documentation) to
+better understand the full process for submitting a new project.
+
+In short, copy this Application Template and populate the document.
+--!>
+
+
+## Background information
+
+1. Submitter Name
+-
+
+2. Submitter’s relationship to project / title
+-
+
+3. Project Name
+-
+
+4. Why this project is valuable to the Kubeflow Community
+-
+
+5. Short Description / Functionality
+-
+
+6. Website
+-
+
+7. GitHub repository
+-
+
+8. 1st Release date
+-
+
+9. Project Meeting Times
+-
+
+10. Meeting Notes (link)
+-
+
+11. Installation Documentation (provide status and link)
+-
+
+12. Application Documentation (provide status and link)
+-
+
+13. Security Profile (CVE scanning)
+-
+
+14. Ownership / Legal Profile (license type, any potential issues for CNCF)
+-
+
+15. Kubernetes native (K8s version)
```

Comment:
We need to explain what do we mean here by K8s native. Does it mean being cloud native/micro-service architecture or something else ?